web/styles/informational_overlays.css

```diff
@@ -1,108 +1,186 @@
 .informational-overlays {
+    display: grid;
+    place-items: center;
+
     .overlay-content {
-        /* because zoom breaks at 525px perhaps due to rounding errors, so add a
-           trivial amount of width so it doesn't break. */
-        width: 550px;
+        width: 90%;
+        max-width: 650px;
         margin: 0 auto;
         position: relative;
-        top: calc((30vh - 50px) / 2);
-        border-radius: 4px;
+        top: unset;
+        border-radius: 6px;
         overflow: hidden;
 
         background-color: var(--color-background-modal);
     }
 
     .overlay-tabs {
-        padding: 10px 0;
-        border-bottom: 1px solid hsl(0deg 0% 0% / 20%);
+        position: relative;
+        display: flex;
+        flex-flow: row nowrap;
```

Comment:
As above, I'd spend time seeing if there are additional properties here that don't need to be declared. Some might have no effect (again, there's typically no need for relative positioning in a grid context; that means you can lose `z-index` too), while some are defaults: `row nowrap` is [the default for `flex-flow`](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-flow#formal_definition). You get them just by setting `display: flex` on an item. Nice, right?

The only time you want to declare default values like that is if you're resetting from another, competing style declaration elsewhere (in which case you'd definitely want to put in a comment saying which style, and why).